conf/messages.cy

```diff
@@ -269,3 +269,10 @@ detailsSelectItem.value.density.litres20 = (temperature of 20&deg;C)
 detailsSelectItem.key.packaging = Packaging
 detailsSelectItem.value.packaging = {0} {1}
 detailsSelectItem.viewAllDetails = View all details for item {0}
+
+chooseShortageExcessItem.title = What''s wrong with item {0}?
+chooseShortageExcessItem.heading = What''s wrong with item {0}?
```

Comment:
I think these should be single curly quotes, which is better typography. This was raised on a previous project by a content writer. Maybe it's something we need to check with Clare and Tom but I've been changing mine.

This was the old article the previous designer sent to me:
https://smartquotesforsmartpeople.com/